rules-tests/TypeDeclaration/Rector/ClassMethod/AddMethodCallBasedStrictParamTypeRector/Fixture/native_array_shape3.php.inc

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\AddMethodCallBasedStrictParamTypeRector\Fixture;
+
+final class NativeArrayShape3 {
+    private function doFoo() {
+        $arr = [];
+        $arr[0] = [123.132];
+        $this->doBar($arr[0]);
+    }
+
+    private function doBar($param) {
+
+    }
+}
+?>
+-----
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\AddMethodCallBasedStrictParamTypeRector\Fixture;
+
+final class NativeArrayShape3 {
+    private function doFoo() {
+        $arr = [];
+        $arr[10] = [123.132];
+        $this->doBar($arr[10]);
```

Comment:
The fixture is copy paste incorrect, the key is `0` :)

```suggestion
        $arr[0] = [123.132];
        $this->doBar($arr[0]);
```

Reply:
args, right.. ;)